sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -279,12 +278,9 @@ def _add_argparse_args(cls, parser):
                         action='store_true',
                         help='Whether to enable streaming mode.')
 
-  # TODO(BEAM-1265): Remove this warning, once at least one runner supports
-  # streaming pipelines.
   def validate(self, validator):
     errors = []
-    if self.view_as(StandardOptions).streaming:
-      warnings.warn('Streaming pipelines are not supported.')
```

Comment:
Should we keep this warning around until at least one runner has support for streaming?